test/e2e/utils.ts

```diff
@@ -36,18 +36,18 @@ export async function map<T>(
 
 type Selector = string | Locator
 
-const toLocator = (page: Page, selector: Selector): Locator =>
-  typeof selector === 'string' ? page.locator(selector) : selector
-
-export async function expectVisible(page: Page, selectors: Selector[]) {
+// note if `Locator` is used instead of string, the container is ignored
+export async function expectVisible(container: Page | Locator, selectors: Selector[]) {
```

Comment:
What do you think of this @david-crespo ?

Alternatively we could remove `Locator` as an option and only use strings. It's only really used in two test files and the change would be trivial.

Reply:
Just use regular `expect().toBeVisible()`. I want to get rid of this utility anyway. I don’t use it in any new code.